src/ctim/schemas/asset_mapping.cljc

```diff
@@ -26,13 +27,26 @@
   c/sourced-object-entries
   (f/required-entries
    (f/entry :type AssetMappingTypeIdentifier)
-   (f/entry :valid_time c/ValidTime)
-   (f/entry :confidence v/HighMedLow)
-   (f/entry :specificity Specificity)
-   (f/entry :stability Stability)
-   (f/entry :observable c/Observable)
-   (f/entry :asset_ref asset/AssetRef)
-   (f/entry :asset_type asset/AssetType)))
+   (f/entry :valid_time c/ValidTime
+            :description (str "For each asset, we allow for the assertion of time bound properties."
+                              "This gives us both a record of the current state of the asset,"
+                              "as well as history."))
+   (f/entry :confidence v/HighMedLow
+            :description "Is it susceptible to manipulation or translation.")
```

Comment:
I would be more general like in [Incident](https://github.com/threatgrid/ctim/blob/f9e2eff38e51ca0c2b8c4099457a1ecf07c8dba3/src/ctim/schemas/incident.cljc#L35) and you could use your description as an example):
"level of confidence held in the characterization of this AssetMapping (e.g. is it susceptible to manipulation or translation?)."